### PR TITLE
Start of ExceptionInstr coverage, cleanup

### PR DESCRIPTION
**Pull request by davidharrishmc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,11 +135,17 @@ export VCS_HOME=..                  # Change this for your path to Synopsys VCS
...
 
-Although most EDA tools are Linux-friendly, they tend to have issues when not installed on recommended OS flavors.  Both Red Hat Enterprise Linux and SUSE Linux products typically tend to be recommended for installing commercial-based EDA tools and are recommended for utilizing complex simulation and architecture exploration.  Questa can also be installed on Microsoft Windows as well as Mac OS with a Virtual Machine such as Parallels.
+Although most EDA tools are Linux-friendly, they tend to have issues when not installed on recommended OS flavors.   Red Hat Enterprise Linux (and its free Rocky clone) typically tend to be recommended for installing commercial-based EDA tools and are recommended for utilizing complex simulation and architecture exploration.
```
Comment (jordancarlin) [inline README.md]: Any particular reason to remove the mention of SUSE from this section? I think all the major EDA vendors support it (at least Questa and Design Compiler are compatible with it).

File: tests/wally-riscv-arch-test/riscv-test-suite/rv32i_m/privilege/src/WALLY-csr-permission-s-01.S [hunk 2]
```diff
@@ -32,18 +32,15 @@ TRAP_HANDLER m
...
 GOTO_S_MODE 0x0, 0x0
 
 # Attempt to write 0x111 to each of these CSRs and read the value back
 # should result in an illegal instruction for the write and read, respectively
 
 # High-bit versions storing the upper 32 bits of some CSRs for RV32
-# WRITE_READ_CSR mstatush 0x111 # *** these appear not to be implemented in GCC
-# WRITE_READ_CSR menvcfgh 0x111
-# WRITE_READ_CSR mseccfgh 0x111
+WRITE_READ_CSR mstatush 0x111  # not supported in rv32
```
Comment (jordancarlin) [inline tests/wally-riscv-arch-test/riscv-test-suite/rv32i_m/privilege/src/WALLY-csr-permission-s-01.S]: Is this comment correct? I assume mstatush is supported in rv32 but not rv64?

**Comment by davidharrishmc** [comment]
Good catches.  Try this again.

**Review by jordancarlin — APPROVED** [review]
Canceled CI because it was just a documentation change

**Pull request merged**